### ISSUE TITLE
Raise hand when sharing screen: Add desktop notification showing who is raising their hand

### ISSUE THREAD

**Issue by hbenali** [jitsi/jitsi-meet#11414]
- **Feature:** Add a desktop notification when someone raises their hand to the person sharing their screen or implement a dedicated JS event for other integrations to make presentation and communication smoother.

- **Actual Behavior:** Whenever someone raises their hand, the presenter should go to the meeting room tab and check who is requesting permission to speak.

Thanks in advance.

**Comment by saghul** [comment]
There is already a notification for raised hands, are you not able to see it?

**Comment by hbenali** [comment]
> There is already a notification for raised hands, are you not able to see it?

Yes, I don't see it, especially when sharing content from another browser tab.

**Comment by saghul** [comment]
Do you see it, if you are on the meeting tab.

It's not very long, but we do show the number of raised hands in yellow at the top and clicking it should take you to the participants pane, so I'm not sure what else we could add that doesn't clutter the UI.

**Comment by hbenali** [comment]
Sounds you completely misunderstood my proposal 😸 thanks for your time.

**Comment by saghul** [comment]
Maybe you can rephrase then?
